server/src/test/java/org/elasticsearch/index/mapper/DateRangeFieldMapperTests.java

```diff
@@ -32,7 +32,7 @@ protected XContentBuilder rangeSource(XContentBuilder in) throws IOException {
 
     @Override
     protected String storedValue() {
-        return "1477872000000";
+        return "1477958399999";
```

Comment:
This is what is expected.  This was the primary bug in this case where values where missing values were essentially being assumed to be 0's rather than appropriately time which in this case is at the very end of the day (Oct 31).  

Mon Oct 31 2016 00:00:00.000
Mon Oct 31 2016 23:59:59.999